src/components/HappySlider/Smileys/smiley1.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Can you adjust the `width` and `height` here

Reply:
That was my first instinct and I did do some minor changes on the SVG files. Originally they all had a height of 25 and I changed that to 24. I didn't dive too deep, though. I was hoping somebody might have some insight on it. I'll dig a little deeper on it. 'viewBox' seems like something that could be useful.

Reply:
I think it's good for now. We'll try to get some feedback from UX next time we can all meet.

Reply:
I think it's fine to alter the svg's themselves. That's the benefit of having them as svg's and not another format, they're scalable ;) we can always easily change them back as well

Reply:
I totally agree. I see no problem in altering the images before they come into the component. I was just curious if we could find a way to import any size image and have it work. 

The background property for the thumb has some odd behavior. If you increase the size, the image does not stay centered, so in this image for example.
![image](https://user-images.githubusercontent.com/54245791/88135365-dc5f3400-cbb4-11ea-82b9-1638e4e15252.png)

If we were to increase the background-size, the image gets larger, but moves askew. Like so.
<img width="104" alt="Screen Shot 2020-07-22 at 12 49 55 AM" src="https://user-images.githubusercontent.com/54245791/88135559-52639b00-cbb5-11ea-9407-28e8347c2f9b.png">
 
Not a huge deal, by any means, but I thought somebody may have run across a similar instance

Reply:
This is helpful: https://css-tricks.com/scale-svg/

Reply:
Oh, this is what you meant! Haha the photo(s) I asked you for on Slack was this 😅